src/browser/pallet.js

```diff
@@ -64,11 +64,9 @@ export const create =
 
 export const requestCuratedColor =
   (uri/*:URI*/)/*:Task<Never, ?Theme>*/ =>
-  Task.future(() => {
+  Task.create(() => {
```

Comment:
We need to keep a promise wrapper here in order to keep an async resolution behavior as it was prior to this change (See https://github.com/browserhtml/browserhtml/pull/1003#issuecomment-211648650 for more details)